libbpf-tools/tcppktlat.c

```diff
@@ -75,6 +91,21 @@ static error_t parse_arg(int key, char *arg, struct argp_state *state)
 	case 'T':
 		env.timestamp = true;
 		break;
+	case 'H':
+		env.histogram = true;
+		if (arg) {
+			errno = 0;
+			env.interval = strtoul(arg, NULL, 10);
```

Comment:
The -H option allows specifying interval both as an optional argument and as a positional argument, which creates ambiguity.